parm/archive/gefs_wave.yaml.j2

```diff
@@ -0,0 +1,36 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + "wave.t" + cycle_HH + "z." %}
+
+gefs_wave:
+    name: "GEFS_WAVE"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_wave.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set file_set = [] %}
+{% set membr = [] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do membr.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% for mem in membr %}
+    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                          '${RUN}':RUN,
+                          '${YMD}':cycle_YMD,
+                          '${HH}':cycle_HH,
+                          '${MEMDIR}':mem }) %}
+
+    {% set COMIN_WAVE_GRID = COM_WAVE_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+    # Select grib2 files to copy to the atardir
+    {% if RUN == "gefs" %}
+        {% if path_exists(COMIN_WAVE_GRID) %}
+            {% for fhr in range(FHMIN, FHMAX_GFS + 1, 1) %}
+                {% set file_name = head ~ "global.5p00.f" ~ '%03d'|format(fhr) ~ ".grib2" %}
```

Comment:
Like for ocean, the grid resolution should not be hard-coded here. The wave product grid will vary depending on the wave model resolution.